src/main/java/de/tum/in/www1/artemis/repository/ProgrammingExerciseStudentParticipationRepository.java

```diff
@@ -29,7 +29,8 @@
 
     @Query("select p from ProgrammingExerciseStudentParticipation p left join fetch p.results pr left join fetch pr.feedbacks left join fetch pr.submission left join fetch pr.assessor"
```

Comment:
Here you could use a text block to get better readability, but I guess that depends on individual tastes.